pkg/mkverification-raw-efi/Dockerfile

```diff
@@ -0,0 +1,73 @@
+# This mkverification-raw-efi produces the raw EFI partition for EVE,
+# including the files in efi-files in the image.  This includes:
+#
+#   /EFI/BOOT/grub.cfg - Chainloads main bootloader
+#   /UsbInvocationScript.txt - Enables USB boot on Dell 3000 series
+#
+FROM lfedge/eve-alpine:145f062a40639b6c65efa36bed1c5614b873be52 AS build
+SHELL ["/bin/ash", "-eo", "pipefail", "-c"]
+ENV BUILD_PKGS mkinitfs grep patch git make coreutils musl-dev gcc g++ perl autoconf automake libtool file bsd-compat-headers libc-dev util-linux-dev flex linux-headers glib-dev libxrandr-dev zlib-dev libusb-dev kmod-dev curl
+ENV PKGS mtools dosfstools libarchive-tools sgdisk e2fsprogs util-linux squashfs-tools coreutils tar dmidecode smartmontools libaio libaio-dev perl glib zlib libusb curl xz pciutils usbutils hdparm util-linux wireless-tools libxrandr eudev-libs libusb sudo
+RUN eve-alpine-deploy.sh
+
+RUN echo -e "http://dl-cdn.alpinelinux.org/alpine/v3.16/main\nhttp://dl-cdn.alpinelinux.org/alpine/v3.16/community\nhttp://dl-cdn.alpinelinux.org/alpine/v3.16/test" > /etc/apk/repositories
+RUN apk add -p /out fio iperf3 sysstat lm_sensors acpi iw libdrm hwinfo dhclient
+RUN apk add --no-cache --virtual build-deps eudev-dev libdrm-dev i2c-tools-dev hwinfo
+
+WORKDIR /out
+RUN echo "mtools_skip_check=1" >> etc/mtools.conf
+RUN mkdir -p efifs parts root bits config persist opt/pillar opt/debug lib/modules run sys
+COPY make-raw verify grub.cfg.in UsbInvocationScript.txt ./
+
+RUN git clone https://git.linuxtv.org/edid-decode.git 2>/dev/null \
```

Comment:
You should use ADD instead of RUN git clone .... to fetch edid-decode repository, and you will not need git as a dependency....

Reply:
I understand your point, but in general it is better to avoid the ADD command. I am not sure that the ADD command can clone a git repo. Unless you suggest to clone the repos in the host and then add them.

Reply:
We want to use ADD and run without network since that makes it possible to generate a SBOM for the build.
We already have examples in Dockerfiles which do various git operations to get the content. @deitch can point you at examples if you can't find them.